xwiki-platform-core/xwiki-platform-menu/xwiki-platform-menu-ui/src/main/resources/Menu/UIExtensionSheet.xml

```diff
@@ -56,12 +56,17 @@
   ## Prepare the extension point map.
   #set ($menuPanel = "{{velocity}}
     ${escapetool.h}panelheader(""$doc.plainTitle"")
-    {{menu type=""vertical collapsible open""}}{{include reference=""${escapetool.d}uix.doc.documentReference"" /}}{{/menu}}
+    {{menu type=""vertical collapsible open""
+    label=""$escapetool.velocity($doc.plainTitle)""}}{{include
```

Comment:
Why do you use `$escapetool.velocity` here? While this might indeed be necessary, this also needs escaping for XWiki syntax.